### PR TITLE
Add 1 case of vm features: kvm poll-control

### PR DESCRIPTION
**Pull request by chloerh**
- Test vm with settings of feature: kvm poll-control

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

**Comment by chloerh** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/2905

**Comment by chloerh** [comment]
测试名称 | 运行时间 | 结果
-- | -- | --
positive_test.hyperv.frequencies | 32 秒 | Passed
positive_test.hyperv.reenlightenment | 28 秒 | Passed
positive_test.hyperv.relaxed.disable | 27 秒 | Passed
positive_test.hyperv.relaxed.enable | 27 秒 | Passed
positive_test.hyperv.tlbflush | 31 秒 | Passed
positive_test.kvm_poll_control.disable | 27 秒 | Fixed
positive_test.kvm_poll_control.enable | 28 秒 | Passed
positive_test.pmu.disable | 28 秒 | Passed
positive_test.pmu.enable | 27 秒 | Passed
positive_test.pvspinlock.disable | 27 秒 | Passed
positive_test.pvspinlock.enable | 28 秒 | Passed

**Comment by chloerh** [comment]
@Yingshun @dzhengfy Could you please review?

File: libvirt/tests/src/cpu/vm_features.py [hunk 1]
```diff
@@ -42,17 +45,16 @@ def run(test, params, env):
...
-            )
+        if kvm_poll_control_attr:
+            if not libvirt_version.version_compare(6, 10, 0):
+                test.cancel('This version of libvirt does not support'
+                            ' kvm poll-control')
+
+        # Set feature attrs
+        test_attrs = [pmu_attr, pvspinlock_attr, kvm_poll_control_attr]
+        [vm_xml.VMXML.set_vm_features(vm_name, **fea_attr)
+         for fea_attr in test_attrs if fea_attr]
```
Comment (Yingshun) [inline libvirt/tests/src/cpu/vm_features.py:57]: Nice, I like it.👍

**Review by Yingshun — APPROVED** [review]
Tested in my environment, all passed

**Comment by chloerh** [comment]
> Tested in my environment, all passed

Thank you for your review and verification :handshake:

**Review by dzhengfy — APPROVED** [review]
LGTM

**Comment by dzhengfy** [comment]
Even though we'v already decided that it is not recommended to check qemu command line in case we can check more within the VM. But I did not see other checking methods in this manual case, so I agree with current method.

**Comment by dzhengfy** [comment]
Even though we'v already decided that it is not recommended to check qemu command line in case we can check more within the VM. But I did not see other checking methods in this manual case, so I agree with current method.

**Pull request merged**